src/app.module.ts

```diff
@@ -1,18 +1,21 @@
-import {Module, ValidationPipe} from '@nestjs/common';
-import {ConfigModule} from '@nestjs/config';
-import {APP_PIPE} from '@nestjs/core';
-import * as Joi from 'joi';
-import {LoggerModule} from 'nestjs-pino';
+import { Module, ValidationPipe } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { APP_PIPE } from '@nestjs/core';
+import { LoggerModule } from 'nestjs-pino';
 import appConfig from './config/app.config';
-import {ReviewsModule} from './reviews/reviews.module';
-import {KafkaModule} from './kafka/kafka.module';
-import {PrismaModule} from './prisma/prisma.module';
+import { ReviewsModule } from './reviews/reviews.module';
+import { KafkaModule } from './kafka/kafka.module';
+import { PrismaModule } from './prisma/prisma.module';
+import { JwtModule } from '@nestjs/jwt';
 
 @Module({
   imports: [
     KafkaModule,
     PrismaModule,
     ConfigModule.forFeature(appConfig),
+    JwtModule.register({
```

Comment:
We should put the secrets in the environment variables, not directly in the code.